### ISSUE TITLE
Assembly instructions missing

### ISSUE THREAD

**Issue by kozross** [clee/VoronBFI#23]
While there is a detailed list of additional required parts, there are no instructions on how to build these.

**Comment by clee** [comment]
Yeah, assembly instructions are on my TODO list, but thanks for making an issue to track it!

**Comment by kozross** [comment]
FWIW, [this video (at the linked timestamp)](https://www.youtube.com/watch?v=XtfESO4ALw8&t=970s) shows the assembly process. If you don't think it's wrong somehow, you could link to it in the interim?

**Comment by cjohnson-phys** [comment]
I think I am most confused about the belt clamps when upgrading from the default v2.4 belt clamps. Any pointers?

**Comment by roykrikke** [comment]
Any status update?

**Comment by messymouse** [comment]
I made an exploded view for BZI, which is pretty much all the instructions needed for these. I don't know how to do the fancy PDF files (I would love to learn!) but I hope this helps =)

![BZI](https://github.com/clee/VoronBFI/assets/139911516/2196dd32-6067-49f4-94df-d822a61e74c0)
